src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -156,9 +156,12 @@ export const getIpIsNotNetworkOrBroadcastAddressSchema = (
     (value) => {
       const subnetAddr = getAddressObject(subnet, protocolVersion);
       if (subnetAddr) {
-        const subnetStart = subnetAddr?.startAddress().correctForm();
-        const subnetEnd = subnetAddr?.endAddress().correctForm();
-        return !(value === subnetStart || value === subnetEnd);
+        if (subnetAddr === null) return true;
```

Comment:
You essentially have the same condition twice here (on line 158 and 159). The second one will probably never evaluate true.

Also, I suppose in case `subnetAddr` is null return value should be `false`?

Reply:
>Also, I suppose in case subnetAddr is null return value should be false?
Exactly, I believe that too. Based on what the Yup [API says](https://github.com/jquense/yup#schematestname-string-message-string--function--any-test-function-schema:~:text=a%20validation%20function%20that%20must%20return%20true%20when%20the%20current%20value%20is%20valid)

Reply:
With these changes works well:
 if (!subnetAddr) {
        return false;
      } else {
        const subnetStart = subnetAddr?.startAddress().correctForm();
        const subnetEnd = subnetAddr?.endAddress().correctForm();
        return !(value === subnetStart || value === subnetEnd);
      }